tests/cli_batch.test

```diff
@@ -0,0 +1,53 @@
+# test for the batch mode with verbose flag
+<
+
+$ ./spellcheck -v tests/test_short.txt
+>
+tests/test_short.txt has been accepted as the input file for target file!
+Dictionary Successfully Parsed!
+
+
+============================================================
+=================== Editing Started With ===================
+============================================================
+
+file: tests/test_short.txt
```

Comment:
This has been removed as well, due to this information being unnecessary for the average user.